plugins/scaffolder/report-alpha.api.md

```diff
@@ -367,6 +362,11 @@ export const scaffolderTranslationRef: TranslationRef<
     readonly 'ongoingTask.hideLogsButtonTitle': 'Hide Logs';
     readonly 'ongoingTask.showLogsButtonTitle': 'Show Logs';
     readonly 'templateEditorForm.stepper.emptyText': 'There are no spec parameters in the template to preview.';
+    readonly 'renderSchema.undefined': 'No schema defined';
```

Comment:
This is technically a breaking change to the translations which I don't think we've come across before.

Wondering what the best practice here is @backstage/maintainers?

Should we call this out in a separate changeset that the keys have moved?